src/frontend/komponenter/Fagsak/Brevmottaker/BrevmottakerContext.tsx

```diff
@@ -68,6 +168,20 @@ const [BrevmottakerProvider, useBrevmottaker] = createUseContext(
             }
         }, [brevmottakere]);
 
+        React.useEffect(() => {
+            if (brevmottakerTilEndring) {
+                const opprinneligInput = brevmottakere[brevmottakerTilEndring];
+                populerSkjema(skjema, opprinneligInput);
+                settAdresseKilde(
+                    opprinneligInput.personIdent
+                        ? AdresseKilde.OPPSLAG_REGISTER
+                        : opprinneligInput.organisasjonsnummer
+                        ? AdresseKilde.OPPSLAG_ORGANISASJONSREGISTER
+                        : AdresseKilde.MANUELL_REGISTRERING
+                );
+            }
```

Comment:
Her trengte jeg et par forsøk for å forstå hva som skjer, lurer på om vi kanskje kan gjøre noe med navngivingen... `brevmottakerTilEndring` er kun en id, sant? Ikke et fullt brevmottakerobjekt. `opprinneligInput` er litt tvetydig på om det kommer fra lagrede verdier backend eller tidligere skjematilstand. 
Forslag: `brevmottakerTilEndring` -> `brevmottakerIdTilEndring` og `opprinneligInput` -> `lagretBrevmottaker`